pyproject.toml

```diff
@@ -129,6 +130,10 @@ no_implicit_optional = true
 check_untyped_defs = true
 warn_return_any = true
 
+[[tool.mypy.overrides]]
+module = ["requests.*"]
+ignore_missing_imports = true
```

Comment:
I think this should be solved by using `types-requests` right? (https://github.com/python/typeshed/tree/main/stubs/requests), [pypi](https://pypi.org/project/types-requests/)

Reply:
I was pretty sure I already did it. Will change it in a sec

Reply:
I was pretty sure I already did it. Will change it in a sec